CDAlertView/Classes/CDAlertView.swift

```diff
@@ -290,8 +294,18 @@ open class CDAlertView: UIView {
         popupViewInitialFrame = popupView.frame
 
         completionBlock = completion
+        
+        if(self.autoHideTime != nil) {
```

Comment:
Sorry for not mentioning at first but I just realized that we're not using parentheses in `if` cases, could you remove that one to have same style in everywhere. Also could you change it to optional unwrapping like
`if let autoHideTime = self.authoHideTime { ...  }` Than, I'll test and merge

Reply:
👍